### PR TITLE
FIX: hotfix for chart detail deleted user info

### PR DESCRIPTION
**Pull request by vikramdevtron**
# Description

Error on charts/apps updated by a user which is marked inactive 

Fixes # (issue) : https://github.com/devtron-labs/devtron/issues/766

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Chart detail fetch after deployed user delete by admin.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: pkg/user/UserService.go [hunk 1]
```diff
@@ -975,3 +976,16 @@ func (impl UserServiceImpl) IsSuperAdmin(userId int) (bool, error) {
 	}
 	return isSuperAdmin, nil
 }
+
+func (impl UserServiceImpl) GetByIdIncludeDeleted(id int32) (*bean.UserInfo, error) {
```
Comment (manish-agrawal-ai) [inline pkg/user/UserService.go:980]: This method is giving only userinfo. 
while getById gives roles and other data as well along with userInfo. 

I hope this is intentional and would not have any impact

**Review by manish-agrawal-ai — COMMENTED** [review]
check comment

File: pkg/user/UserService.go (hunk 1 above)
Comment (vikramdevtron) [reply pkg/user/UserService.go:980]: yes this was intentional, we required only email for the service.

Comment (manish-agrawal-ai) [reply pkg/user/UserService.go:980]: ok

**Review by manish-agrawal-ai — APPROVED** [review]

**Pull request merged**